Lib/pathlib/__init__.py

```diff
@@ -1140,11 +1144,20 @@ def _copy_from_file(self, source, preserve_metadata=False):
         _copy_from_file_fallback = _copy_from_file
         def _copy_from_file(self, source, preserve_metadata=False):
             try:
-                source = os.fspath(source)
+                source_fspath = os.fspath(source)
             except TypeError:
                 pass
             else:
-                copyfile2(source, str(self))
+                try:
+                    copyfile2(source_fspath, str(self))
+                except OSError as exc:
+                    winerror = getattr(exc, 'winerror', None)
+                    if (_winapi is not None and
+                        winerror in (_winapi.ERROR_PRIVILEGE_NOT_HELD,
+                                     _winapi.ERROR_ACCESS_DENIED)):
+                        self._copy_from_file_fallback(source, preserve_metadata)
+                        return
+                    raise
```

Comment:
```suggestion
                try:
                    copyfile2(source_fspath, str(self))
                except OSError as exc:
                    # Use fallback on ERROR_ACCESS_DENIED
                    # or ERROR_PRIVILEGE_NOT_HELD 
                    if exc.winerror not in (5, 1314):
                        raise
                else:
                    return
```

This code is Windows-specific because of the `if copyfile2:` line above, so we can access `exc.winerror` directly I think.

We usually hard-code Windows error numbers rather than using `_winapi` in pathlib - I'm not sure if that's wise but I guess let's be consistent.

Reply:
Thanks—good call. I switched to exc.winerror and check for 5/1314. If we hit either, we fall back to _copy_from_file_fallback; otherwise we re-raise. I also added a tiny comment mapping those to ERROR_ACCESS_DENIED and ERROR_PRIVILEGE_NOT_HELD. Still under the if copyfile2: guard, so Windows-only. Happy to tweak wording or style if you prefer.